overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -131,8 +131,7 @@ func queueCommand(context *hookstate.Context, tts []*state.TaskSet) error {
 
 func runServiceCommand(context *hookstate.Context, inst *servicestate.Instruction) error {
 	if context == nil {
-		// this message is reused in health.go
-		return fmt.Errorf(i18n.G("cannot %s without a context"), inst.Action)
+		return createMissingContextError(inst.Action)
```

Comment:
It'd be nice to add a unit test checking the error for stop/start/restart.

Reply:
Added.